vp/workflow/views.py

```diff
@@ -1,64 +1,73 @@
 from django.http import JsonResponse
+from .models import Workflow, WorkflowException
 import networkx as nx
 import json
 import os
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
For when we get around to it, Django has a [file storage API](https://docs.djangoproject.com/en/3.0/topics/files/) for this purpose. We can point it to `/tmp` or elsewhere to avoid writing to the repo, and the wrapper guarantees safe storage and access no matter where it's deployed.

Reply:
👍 Added to [the Trello card](https://trello.com/c/bwU6NRXG) for future reference